ups/ci_hsc.table

```diff
@@ -3,16 +3,13 @@ setupRequired(log)
 setupRequired(meas_astrom)
 setupRequired(meas_extensions_psfex)
```

Comment:
You should make an explicit `meas_algorithms` requirement in here, since that's now an explicit dependency.